code-challenges.test.js

```diff
@@ -34,6 +35,41 @@ const object3 = { number: -7 }
 
 // b) Create the function that makes the test pass.
 
+describe ("math",() => {
+    it("return answer", () => {
+        expect(math(object1.number)).toEqual("15 is divisible by three")
+        expect(math(object2.number)).toEqual("0 is divisible by three")
+        expect(math(object3.number)).toEqual("-7 is not divisible by three")
+    })
+})
+
+
+
+
+
+//psuedo code: 
+// create a test function wiht expect funcitons.
+//name the test using descrive 
```

Comment:
I love the pseudo code you're writing this week, Ney! I can see a difference in it from last assessment. Keep it up!